Makefile

```diff
@@ -172,7 +174,7 @@ help:
 	@echo "to operate on subset of components. Example: make COMPONENTS=\"gui\" get-sources"
 
 
-get-sources-sort = $(BUILDER_PLUGINS) $(filter-out builder $(BUILDER_PLUGINS), $(COMPONENTS))
+get-sources-sort = $(BUILDER_PLUGINS) $(filter-out $(BUILDER_PLUGINS), $(COMPONENTS_NO_BUILDER))
```

Comment:
Actually I think `builder` shouldn't be excluded here. See QubesOS/qubes-issues#1405. But that worth a separate commit :)